doc/api/esm.md

```diff
@@ -340,11 +340,11 @@ module default import or its corresponding sugar syntax:
 
 <!-- eslint-disable no-duplicate-imports -->
 ```js
-import { default as cjs } from 'cjs';
+import { default as cjs } from 'node:cjs';
```

Comment:
This seems wrong to me after looking at it again. There is no core module named `cjs`.

Reply:
Thanks I'll be sure to fix this.

Reply:
@DerekNonGeneric I fixed this, and a couple other instances of `node:cjs`.

Reply:
/cc @nodejs/modules if anyone knows why the code blocks here have a `cjs` specifier